### PR TITLE
Revert: Rollback folder check

### LINKED ISSUE #310: Check if source folder exists and exit with helpful error message
**Describe the solution you'd like**
Inspired by #308, where a previous step in a workflow didn't error out, even if the source folder was not create, a less helpful error message was returned by rsync. 

The action could help by checking if the source folder exists before calling rsync, and provide a helpful error message instead, if it doesn't.

### PR DESCRIPTION
**Pull request by exuanbo**
Fixes #310

**Description**
> Provide a description of what your changes do.

Revert commit that rollbacked folder check.

Integration tests passed on my fork https://github.com/exuanbo/github-pages-deploy-action/commit/63b77fb9a99fdd7915f2ceca546c957ee8cf10f2.

### CODE REVIEW

**Comment by exuanbo** [comment]
For some reason it works for me. Check these [commits on my dev branch](https://github.com/exuanbo/github-pages-deploy-action/commits/dev).

**Comment by JamesIves** [comment]
Appreciate the pull request, I will review this shortly.

File: src/util.ts [hunk 1]
```diff
@@ -53,6 +54,12 @@ export const hasRequiredParameters = (action: ActionInterface): void => {
       "Incorrectly formatted build folder. The deployment folder cannot be prefixed with '/' or './'. Instead reference the folder name directly."
     )
   }
+
+  if (!existsSync(action.folder) && action.folder !== action.root) {
```
Comment (JamesIves) [inline src/util.ts:58]: I'm thinking this may need to be reworked slightly with the recently merged absolute folder paths. What do you think?

**Comment by exuanbo** [comment]
This branch seems to be messed up by me...I'll create another pr.

**Comment by JamesIves** [comment]
Looks like it's out of sync with what's in the dev branch. You may be able to solve it by rebasing with dev, or by closing the branch and opening it again. I think there needs to be some changes to this to make it work with the absolute folder paths otherwise the check will be performed incorrectly.

**Comment by JamesIves** [comment]
Lemme know if you plan on re-opening this, if not I will work on this when I have a chance

**Comment by exuanbo** [comment]
Yes I will open another pr in these two days.

**Pull request closed without merging**